tools/operator-upgrade.sh

```diff
@@ -212,7 +212,7 @@ deploy_operator() {
     echo "Operator installation not ready..."
     exit 1
   fi
-  echo "deployment of new splunk operator 1.1.0 complete"
+  echo "deployment of new Splunk Operator 1.1.0 complete"
```

Comment:
Shouldn't it be 2.0.0 instead of 1.1.0 here? 1.1.0 is also on lines 60, 68, 203

Reply:
+1